server/views/pages/tagsSeries.html

```diff
@@ -0,0 +1,97 @@
+{% from "../components/related-content/macro.njk" import hubRelatedContent %}
+{% from "../components/feedback-widget/macro.njk" import hubFeedbackWidget %}
+{% from "../components/page-navigation/macro.njk" import pageNavigation %}
+{% from "../components/user-details/macro.njk" import userDetails %}
+
+{% extends "../components/template.njk" %}
+
+{% block pageTitle %}
+  {{ title }}
+{% endblock %}
+
+{% block pageNavigation %}
+  {{ pageNavigation({ title: title }) }}
+{% endblock %}
+
+{% block topBar %}
+  {{ topBar({
+    establishmentDisplayName: establishmentDisplayName,
+    showBar: false,
+    userName: config.userName,
+    returnUrl: config.returnUrl
+  }) }}
+{% endblock %}
+
+{% block userDetails %}
+  {{ userDetails({
+    detailsType: config.detailsType,
+    userName: config.userName
+  }) }}
+{% endblock %}
+
+{% block header %}{% endblock %}
+{% block homeNavigation %}{% endblock %}
+{% block search %}{% endblock %}
+
+{% block head %}
+  <link href="/public/stylesheets/application.css" rel="stylesheet"/>
+{% endblock %}
+
+{% block main %}
+  <div id="category-{{tagId}}" class="govuk-hub-landing-page-header govuk-hub-landing-page-header--with-stripe">
+    <div class="govuk-width-container">
+      <div class="govuk-main-wrapper">
+        <div class="govuk-grid-row">
+          <div class="govuk-grid-column-one-half govuk-!-padding-top-2">
+            <h1 id="title" class="govuk-heading-xl">{{ data.name }}</h1>
+
+            {% if data.description.summary %}
+              <p id="summary" class="govuk-body-l">{{ data.description.summary}}</p>
+            {% endif %}
+
+            {% if data.description.sanitized %}
+              <div id="description" class="gov-uk-dynamic-content">
+                {{data.description.sanitized | safe }}
+              </div>
+            {% endif %}
+          </div>
+          {% if data.image.url %}
+            <div class="govuk-grid-column-one-half">
+              <div data-page-featured-image class="govuk-hub-tag-page-header__content-image" style="background-image: url({{data.image.url}});"></div>
+            </div>
+          {% endif %}
+        </div>
+      </div>
+    </div>
+  </div>
+
+{% endblock %}
+
+{% block content %}
+  {{ hubRelatedContent({
+    id: tagId,
+    data: data.relatedContent,
+    lazyLoadContent: true
+  }) }}
+
+  <div class="govuk-form-group govuk-!-margin-bottom-2">
+    {% if data.contentType =='series'%}
+      <p class="govuk-hub-article-feedback govuk-hub-feedback-text">Tell us what you think about this series:</p>
+    {% endif %}
+
+    {% if data.contentType =='tags'%}
```

Comment:
guess this isn't needed any more, but NBD. We'll hopefully revisit this as part of the next card